src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -110,7 +109,29 @@ paths:
               schema:
                 type: integer
                 format: int32          
+              
+              
 
+  /recordTree/{id}:
+    get:
+      tags:
+        - '${project.name}'
+      summary: 'Read a specific record by ID. Will return null if no record with the given ID exists. A record marked with delete flag will also be returned. Do NOT call toString() on the object return due to OpenAPi infinite loop'
```

Comment:
Please describe what a record tree is and what happens in the two cases (record is a parent vs. record is a child).

Reply:
also rewritten